shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/ManagerSyncServiceImpl.java

```diff
@@ -81,6 +87,10 @@ protected Uni<Object> doBridges() {
                                         },
                                         failure -> failedToSendUpdateToManager(y, failure));
                     }
+                    if (y.getStatus().equals(ManagedResourceStatus.DELETING)) { // Bridges that were being deleted (before the Operator failed).
+                        LOGGER.info("Found Bridge '{}' in DELETING state. Attempting to continue with DELETING.", y.getId());
+                        return Uni.createFrom().voidItem().subscribe().with(success -> deleteBridgeIngress(y));
```

Comment:
is there any specific reason for `Uni.createFrom().voidItem().subscribe()...`? 
Actually in this case you can 
```java
deleteBridgeIngress(y);
return Uni.createFrom().voidItem();
```

same for the other cases

Reply:
Yes, `Uni.createFrom().voidItem().invoke(()->...)` didn't actually run. I could only get the code to run by subscribing to it.

Reply:
if you use 
```
deleteBridgeIngress(y);
return Uni.createFrom().voidItem();
```
it should work, can you try?

Reply:
@r00ta that worked too. I'll make the change.